README.md

```diff
@@ -80,4 +80,9 @@ godot, where `XXXX` stands for the desired size.
     - And a `godot-bootstrap.jar` in `build/libs` of the root `godot` folder
 4. export LD_PRELOAD=<path_to_sample_project>/jre/lib/libjsig.so
 5. scons -j20 platform=<platform> module_mono_enabled=yes
-6. run with `../../../../bin/godot.x11.tools.64.mono`
\ No newline at end of file
+6. run with `../../../../bin/godot.x11.tools.64.mono`
+
+## Partners
+Jetbrains is helping us to develop this project by providing development tools to maintainers.  
+We also recommend Jetbrains Intellij' Idea to get into kotlin development.  
```

Comment:
I would rewrite this sentence to: `Intellij Idea is our IDE of choice for kotlin development and we strongly recommend using it.`